Dockerfile.postgres

```diff
@@ -0,0 +1,37 @@
+FROM python:3.9 AS decrypt
+
+COPY ./secrets /secrets
```

Comment:
lets move this after ansible install

Reply:
because this way each time you change ./secrets file you need to rebuild everything